packages/shared/src/components/post/PostContent.tsx

```diff
@@ -108,7 +108,23 @@ export function PostContent({
           className={className?.fixedNavigation}
         />
       )}
-      <PostContainer className={classNames('relative', className?.content)}>
+
+      <PostContainer
+        className={classNames('relative', className?.content)}
+        data-testid="postContainer"
+      >
+        {!hasNavigation && (
+          <PostModalActions
```

Comment:
More generic comment, maybe we should consider renaming it, as it's not particular related to modal anymore.

Reply:
I also see we already load this component in PostNavigation is it maybe not as simple as allowing it to be enabled from there? (I'm not quite sure they share the same placement though)

As technically the modal and page share the same content it should be available already.

Reply:
@sshanzel 

> Also, kindly check if this shows on post page for squad post where there is another navigation header.

Thanks for pointing this out. This looks good on the squad post page -- as far as I can tell the squad posts actually use a different component entirely to render (the new header is only added to the `PostContent` component
 https://github.com/dailydotdev/apps/blob/06978d27eada1f40d3d826ecfa4cfdb51e40f39e/packages/webapp/pages/posts/%5Bid%5D/index.tsx#L71-L74

Below is a preview as well:

<img width="382" alt="Screenshot 2023-08-01 at 13 49 13" src="https://github.com/dailydotdev/apps/assets/1225100/71283db8-dd0d-4573-b7c4-4361b106b45f">

---

> One area we can probably look at as well is the navigation variable in the post page to ensure it would only show to specific post types, in this case, the article only. Would also allow us to avoid using the hasNavigation as it would only be applied on the post page.

There is just one caveats here:
- related to the props `PostModalActions` expects to be passed in. We already have all the variables/functions to pass defined within the `PostContent` component (they are passed to the `PostWidget` component as well). So if we decide to move this further up the tree, we would need to redefine those which would create some duplication as well as mixing up responsibilities a bit. 

If you feel that is an acceptable tradeoff, I can go ahead and change it to use the navigation variable.

Reply:
Good point. We will be pulling a lot of hooks and callbacks just to satisfy having it in the `navigation` variable. Either of the two is okay to me. Would love to hear what others think of it too. cc @rebelchris @capJavert @TomDowling 

Context: we have a variable in the post page file named `navigation` and it holds the value for which kind of navigation to render that is processed within the components inside the `PostContent`. The key difference here is that using the `navigation` would keep all of our navigations in the same place, easier to manage, but it would require pulling a lot of hooks to satisfy the requirement of the actions we can perform on the component to render (PostModalActions) - which on the current implementation does not require any additional hook instances since the callbacks are readily available to where it is currently rendered. There are pros and cons on both ends but would love to hear more thoughts.

Reply:
Agree with Ante, great point for down the line improvement, but then again the whole render could do with a revamp 👍